pkg/metrics/metrics.go

```diff
@@ -0,0 +1,141 @@
+package metrics
+
+import (
+	logging "github.com/op/go-logging"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	subsystem = "asb"
+)
+
+var (
+	sandboxCreated = prometheus.NewCounter(
```

Comment:
Why do we want to count all the namespaces created? To me this metric means 'the total number of actions the broker has performed'.

Reply:
I think sandbox is specific, this will give us some nice indirection metrics, say if we have a lot of sandbox creations and no sandbox deletions then things are going very wrong somewhere. This is not the actions, bind and unbind are configured by default to not launch APB's and therefore will not be counted in this metric.

Reply:
> I think sandbox is specific, this will give us some nice indirection metrics, say if we have a lot of
> sandbox creations and no sandbox deletions then things are going very wrong somewhere.
>
It's a good metric to be aware that the sandboxes are being cleaned up, but we should do that with a gauge instead of two counters.

> This is not the actions, bind and unbind are configured by default to not launch APB's and
> therefore will not be counted in this metric.
>
We can still count them if you increment in the if block. 
https://github.com/openshift/ansible-service-broker/blob/81381b1600c489a193fd13afb0a4862ed7b878fa/pkg/broker/broker.go#L865